internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -561,6 +583,10 @@ func (r *Reconciler) generateRepoHostIntent(postgresCluster *v1beta1.PostgresClu
 		repo.Spec.Replicas = initialize.Int32(1)
 	}
 
+	// Set the StatefulSet Pod Management Policy to "Parallel" so that the StatefulSet can
+	// recreate the pod, if it initially has an issue (like an bad image).
+	repo.Spec.PodManagementPolicy = appsv1.ParallelPodManagement
```

Comment:
⛏️ I like the links to `#rolling-updates` and `#forced-rollback` in the other files. I'd also like to see them here, but it's not a blocker.

⛏️ Similarly, I'd like to see the `UpdateStrategy.Type` set here to clarify that the STS controller will roll out pod changes.